lib/l4p/udp_rxtx.c

```diff
@@ -108,6 +108,33 @@ rx_stream(struct tle_udp_stream *s, void *mb[], struct rte_mbuf *rp[],
 	return r;
 }
 
+uint16_t tle_udp_stream_rx_bulk(struct tle_stream *us, struct rte_mbuf *pkt[],
+		struct rte_mbuf *rp[], int32_t rc[], uint16_t num)
+{
+	struct tle_udp_stream *s;
+	uint32_t i, n;
+
+	s = UDP_STREAM(us);
+	if (s == NULL) {
+		for (i = 0; i != num; i++) {
+			rc[i] = ENOENT;
+			rp[i] = pkt[i];
+		}
+		return 0;
+	}
+
+	n = rx_stream(s, (void*)pkt, rp, rc, num);
+
+	if (s->rx.ev != NULL)
+		tle_event_raise(s->rx.ev);
+	// unclear what purpose this serves since we arent acquiring a stream
```

Comment:
I'd remove the comments here, I'm assuming this was copied from TCP where there is an acquire somewhere?

Reply:
this was based on `tle_udp_rx_bulk` https://github.com/Trinity-Cyber-Inc/tldk/blob/1dac0c24f33738f7f31c2ce4cb0bf98384f6954c/lib/l4p/udp_rxtx.c#L195